docker/my_init.d/81-recompile-assets

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# recompile assets at docker instance boot time
```

Comment:
If you look at opg-core-docker, we can use the makefile to compile the assets without installing the build tools in the container.